tasks.py

```diff
@@ -251,15 +251,15 @@ def list(self, *args, **kwargs):
 		"""Lists all user-callable tasks which are callable by this user"""
 		global _callableTasks
 
-		class extList(list):
-			pass
-
-		res = extList(
-			[{"key": x.key, "name": str(x.name), "descr": str(x.descr)} for x in _callableTasks.values() if
-			 x().canCall()])
-		res.cursor = None
-		res.baseSkel = {}
-		return (self.render.list(res))
+		tasks = db.SkelListRef(None)
```

Comment:
Just replace this with `tasks = db.SkelListRef({)`  - then there is no need to mess with the render :)

Reply:
I tried this solution but it causes another error, because task does not have a skelStructure. What can we do to mitigate this? I causes an error in the json renderer, so we are back to phorwards issue

Reply:
Hm - can you provide a traceback? I've tested it locally and that seemed to fix the issue.

Reply:
I can confirm @tsteinruecken's answer, it now works. 

Anyway, I have a proposal for another improvement made in just another commit 4cc94a29dd2fe018a786b27d31e1a266f622de9d. It makes SkelList's baseSkel default to an empty dict, if not provided.

@xnopasaranx can you please provide more details on your problem?

Reply:
Works for me as well :)